source/dub/commandline.d

```diff
@@ -2281,6 +2281,7 @@ class DustmiteCommand : PackageBuildCommand {
 		bool m_combined;
 		bool m_noRedirect;
 		string m_strategy;
+		uint m_jobCount = uint.min; // zero means not specified
```

Comment:
why is this set to `uint.min`?

Reply:
Ahh, it's really not needed. I'll remove.

Reply:
Done.

Reply:
You dan't get notifications for edit, but as long as you are either subscribed to notifications to the repository, participate in, or are mentioned either directly or requested for review you get an email.

Take a look at the footer of email threads from GitHub and you'll see things like `You are receiving this because you were mentioned.` or `You are receiving this because you are subscribed to this thread` or some such.

As for the delay in reply. Timezones are a thing, and people need to sleep ;)

Reply:
I see. Thanks! :)